lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -141,6 +146,10 @@ def command=(value)
     task.application_name = value
   end
 
+  def comment=(value)
+    task.comment = value
```

Comment:
Ok, so we'll leave the length checks to a seperate PR (but I cannot promise that I'm volunteering for that, if you expected that anyway ;-).